e2e-tests/micronaut-rest-api/src/test/java/org/evomaster/e2etests/micronaut/rest/MicronautTest.java

```diff
@@ -0,0 +1,53 @@
+package org.evomaster.e2etests.micronaut.rest;
+
+import com.foo.micronaut.rest.MicronautTestController;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.evomaster.core.problem.rest.HttpVerb;
+import org.evomaster.core.problem.rest.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.e2etests.utils.RestTestBase;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import static org.hamcrest.core.Is.is;
+
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MicronautTest extends RestTestBase {
```

Comment:
add comment that this is the same as `TcpPortEMTest` but done for Micronaut.
actually, could also rename with: `TcpPortMicronautEMTest`